examples/node/spec/unit/credential.spec.ts

```diff
@@ -3,7 +3,9 @@ import nock from "nock";
 import Twilio from "../../lib/rest/Twilio";
 
 describe("credential", () => {
-  const twilio = new Twilio();
+  const accountSid:string = 'AC12345678123456781234567812345678';
+  const authToken:string = 'CR12345678123456781234567812345678';
+  const twilio = new Twilio(accountSid, authToken);
```

Comment:
`username` and `password` are optional so can revert these.

Reply:
extending BaseTwilio makes it so the username and password were required